### ISSUE TITLE
Problem playing media files

### ISSUE THREAD

**Issue by szwart** [NASA-Tournament-Lab/NTL-ISS-Food-Intake-Tracker#184]
After pulling a report and clicking on one of Saint-Jacques' media files (20172102192609), I get a message that comes up indicating that Windows Media Player encountered a problem while playing the file.  I was able to play the next file (ending in 850), but not the one ending in 805.

**Comment by coderReview** [comment]
It seems that the file is very small. Meaning it didn't record anything. Can you try recording a new file for a few seconds a try report again? cc @nithyaasworld

**Comment by rsial2** [comment]
@szwart can you try again today please?

**Comment by szwart** [comment]
Absolutely. In a meeting now but will try again later.

On Mar 3, 2017, at 9:47 AM, Rashid Sial <notifications@github.com<mailto:notifications@github.com>> wrote:


@szwart<https://github.com/szwart> can you try again today please?

**Comment by szwart** [comment]
I added a few voice files to the Nick Hague profile where I counted to 10.  I'm still getting an error indicating the "Windows Media Player encountered an issue while playing the file".

**Comment by szwart** [comment]
No… I click on the food record to open details and when I click the play button a grey bubble pops up and says “Player stopped”.

**Comment by szwart** [comment]
Ah – I figured out what it was.  In the settings, the app access to microphone was not toggled on.  It’s working now.

**Comment by coderReview** [comment]
@szwart great news. If you want I can add this information to the documentation. @rsial2 what do you think?

**Comment by rsial2** [comment]
Yes please add it @coderReview. We'll close this ticket once it's been added.

**Comment by szwart** [comment]
Sounds good.

**Comment by coderReview** [comment]
@rsial2 documentation was updated. Please check it out.
